### ISSUE TITLE
Issue template testing

### ISSUE THREAD

**Issue by consideRatio** [ThoriumEnergyWorld/TMSR-Course#2]
## Is this issue lecture specific?
- [ ] 0 - Course Introduction and Build-up
- [ ] 1 - Introduction to Thorium Energy and the TMSR 
- [ ] 2 - TMSR History, Reactor Basics and Developers
- [ ] 3 - Fuel cycle aspects of the TMSR
- [ ] 4 - Neutronics of the TMSR
- [ ] 5 - Thermal-Hydraulics and Fluid Dynamics (CFD) for the TMSR
- [ ] 6 - TMSR Simulations
- [ ] 7 - Control Strategies of the TMSR
- [x] 8 - Kinetics, dynamics and noise analysis of the TMSR
- [ ] 9 - Thermodynamics and physicochemical properties of salts in the TMSR
- [ ] 10 - Materials and Metals in the TMSR
- [ ] 11 - Salt Chemistry, Pyrochemical methods for fuel salt reprocessing and chemical processing in the TMSR
- [ ] 12 - Safety analysis of the TMSR
- [ ] 13 - Challenges

**Comment by consideRatio** [comment]
@ASamarkRoth I made a template like this one in order to ensure the issue submitter clarifies if it is a certain lecture that the issue relates to, then someone like me who will read all issues can label and ping related people that does not read all issues.

I also created a label for each lecture, to make it easy to search and navigate. BTW I just googled if it was possible to subscribe to labels, it was not. Their main purpose would be for filtering and organizing etc.
